src/pulse3D/plate_recording.py

```diff
@@ -591,7 +595,7 @@ def to_dataframe(self) -> pd.DataFrame:
             data[well_name] = pd.Series(interp_force_newtons_normalized)
 
             # add stim data
-            if output_stim_data:
+            if output_stim_data or include_stim_data:
```

Comment:
Here we just want to check if any stim data is even present. You can probably check `aggregate_stim_timepoints_us_for_plotting` somehow to figure that out